packages/react/rollup.config.js

```diff
@@ -51,6 +51,7 @@ const cjsPlugins = [
 const isExternal = (id) => {
   return (
     id !== './FronteggProvider' &&
+    id !== './AlwaysRenderInProvider' &&
```

Comment:
why?

Reply:
We need to add every new file to include it in the bundle :(